src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -110,4 +115,30 @@ def test_email_sent(self):
 
         email = mail.outbox[0]
         self.assertEqual(email.to, ["hello@open-forms.nl"])
-        self.assertEqual(email.subject, _("Your form submission"))
+        self.assertEqual(
+            email.subject, _(f"Opgeslagen formulier {submission.form.name}")
+        )
+
+        self.assertIn(submission.form.name, email.body)
+        self.assertIn(timezone.now().date().strftime("%d-%m-%Y"), email.body)
```

Comment:
please check this for correct timezone activation and use `@freeze_time` in the test so that we can assert literal values/strings in the test rather than a programmatic construction which is just a re-implementation of the code you're testing.